doc/nrf/ug_matter_overview_device_types.rst

```diff
@@ -0,0 +1,210 @@
+
+Sensor device types
+===================
+
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Device type       | Description (from Device Library specification)                              | State of the device | Dedicated sample in the |NCS|         |
++===================+==============================================================================+=====================+=======================================+
+| Contact Sensor    | A Contact Sensor device reports boolean state (open/close                    | Certifiable         |                                       |
+|                   | or contact/no-contact).                                                      |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Light Sensor      | A Light Sensor device is a measurement and sensing device that is capable    | Certifiable         |                                       |
+|                   | of measuring and reporting the intensity of light being emitted              |                     |                                       |
+|                   | by a light source.                                                           |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Occupancy Sensor  | An Occupancy Sensor is a measurement and sensing device that is capable      | Certifiable         |                                       |
+|                   | of measuring and reporting the occupancy state in a designated area.         |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Temperature       | A Temperature Sensor device reports measurements of temperature.             | Certifiable         | :ref:`matter_weather_station_app`     |
+| Sensor            |                                                                              |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Pressure Sensor   | A Pressure Sensor device measures and periodically reports the pressure      | Provisional         | :ref:`matter_weather_station_app`     |
+|                   | of a fluid.                                                                  |                     |                                       |
```

Comment:
I'm not sure about this pressure of a fluid. I can see that this is what the device library document says, but in our case sensor measures atmospheric pressure.

Reply:
The latest Device Library Matter spec defines pressure sensors as follows:
7.5. Pressure Sensor
A Pressure Sensor device measures and reports the pressure of a fluid.

Reply:
No changes made. Should we remove the mention of the weather app in this line if we're not matching the device type description, @kkasperczyk-no , @doublemis1 ?

Reply:
I will keep Weather Station as it is since the fluid is a Liquid or Gas